VERSION

```diff
@@ -0,0 +1 @@
+1.0.0-pre1
```

Comment:
Suggest adding this line https://github.com/tilezen/vector-datasource/blob/master/setup.py#L4 with:

```
version = open("VERSION").read()
```